src/ui/molecules/accordion/index.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react';
+import styled, { StyledComponent } from 'styled-components';
+import { ArrowDown } from 'icons';
+import { Variant } from 'lib/types';
+
+interface AccordionProps {
+  className?: string;
+  isOpen: boolean;
+  title?: React.ReactNode;
+}
+
+const AccordionBase: React.FC<AccordionProps & Variant> = ({
+  children,
+  className,
+  isOpen = true,
+  title,
+  variant = 'secondary',
+}) => {
+  const [isContentVisible, setContentVisible] = React.useReducer((is) => !is, isOpen);
+
+  const onKeyDown = React.useCallback(
```

Comment:
Аккордеон имеет много элементов, смотря на название этой переменной невозможно понять на какой элемент будет добавлен этот хендлер.

Вдруг глобально?

```suggestion
  const onHeaderKeyDown = React.useCallback(
```